sonar-dotnet-shared-library/src/test/java/org/sonarsource/dotnet/shared/plugins/UnitTestResultsProviderTest.java

```diff
@@ -43,8 +43,7 @@ public void vbnet() {
       UnitTestResultsProvider.DotNetUnitTestResultsAggregator.class,
       UnitTestResultsImportSensor.class);
     assertThat(propertyKeys(extensions)).containsOnly(
-      "sonar.vbnet.vstest.reportsPaths",
-      "sonar.vbnet.nunit.reportsPaths");
+      "sonar.vbnet.vstest.reportsPaths", "sonar.vbnet.nunit.reportsPaths");
```

Comment:
@mary-georgiou-sonarsource do you find this to be more readable? personally I find separate lines to be more readable.

Reply:
Exactly. That was my thinking—that there are only two values, and separating them into two lines is not really needed—it does not add much apart from an extra line.
However, if you think it's better to separate not only for readability but also for uniformity with other similar cases, I'll be happy to revert in one of the next PRs.

Reply:
I don't think we should spend too much time on discussing such topics (very low-impact)